wiki/Client/en.md

```diff
@@ -0,0 +1,22 @@
+---
+stub: true  # this could make for a good introductory page
```

Comment:
very sus

```suggestion
stub: true
```

Reply:
while this is valid, as front matter is YAML, I agree it's best read as a TODO